packages/sdk/contractkit/src/wrappers/Election.test.ts

```diff
@@ -146,15 +168,25 @@ testWithGanache('Election Wrapper', (web3) => {
           promises.push(promise)
         }
         await Promise.all(promises)
+      })
 
+      test('SBAT activate vote', async () => {
+        const activeVotes = await election.getActiveVotesForGroup(groupAccount)
+        expect(activeVotes).toEqual(ONE_HUNDRED_GOLD)
+      })
+
+      test('active votes remain unchanged when group becomes ineligible', async () => {
```

Comment:
Should the validator be deactivated in this test?

Reply:
yes good catch!